test/sql-luatest/gh_8676_fix_exists_test.lua

```diff
@@ -0,0 +1,20 @@
+local server = require('luatest.server')
```

Comment:
Please, rename the test file to something like `gh_8676_exists_in_multiselect_test.lua` or something alike. `fix` suffix is redundant - all regression tests are about a "fix".

Reply:
Thanks, fixed.